docs/reference/search/async-search.asciidoc

```diff
@@ -123,9 +123,12 @@ number of new shard responses (`5` by default).
 enforce the execution of a pre-filter roundtrip to retrieve statistics from
 each shard so that the ones that surely don't hold any document matching the
 query get skipped.
-* `ccs_minimize_roundtrips` defaults to `false`. When doing a cross-cluster search,
-setting it to `true` may improve overall search latency, particularly when
-searching clusters with a large number of nodes.
+* `ccs_minimize_roundtrips` defaults to `false`. When doing a {ccs}, setting
+it to `true` may improve overall search latency, particularly when searching
+clusters with a large number of nodes. However, when set to `true`, search
```

Comment:
I see number of nodes being mentioned, but like I mentioned before, the performance is a factor of the number of shards and not the number nodes.

Reply:
ops, seems like I had started a review which I forgot to submit, no wonder why the comment was not addressed :)

Reply:
Thanks. Fixed.